lib/SimpleSAML/Locale/TwigTranslator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Wrap twig trans to allow us to override the translator used.
+ *
+ * @package SimpleSAMLphp
+ */
+
+declare(strict_types=1);
+
+namespace SimpleSAML\Locale;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class TwigTranslator implements TranslatorInterface
+{
+    private $translator;
+
+    /**
+     * @param callable|null $translator
+     */
+    public function __construct(callable $translator = null)
+    {
+        if (!is_callable($translator)) {
```

Comment:
Help me here.. All I can find is cases where we call this with a callable, never with the null-case..
Can we drop this if-statement / the nullable-param?

Reply:
Sure, the check is here to mimic the old behaviour - but as we always provide the translator I see no reason not to drop it.

Reply:
Great, thanks!